src/plugins/discover/public/plugin.tsx

```diff
@@ -188,10 +188,11 @@ export class DiscoverPlugin
   private docViewsRegistry: DocViewsRegistry | null = null;
   private stopUrlTracking: (() => void) | undefined = undefined;
   private locator?: DiscoverAppLocator;
+  private isDev = false;
 
   setup(core: CoreSetup<DiscoverStartPlugins, DiscoverStart>, plugins: DiscoverSetupPlugins) {
     const baseUrl = core.http.basePath.prepend('/app/discover');
-
+    this.isDev = this.initializerContext.env.mode.dev;
```

Comment:
since `this.isDev` is just needed in the setup function it could just be a

`const isDev = this.initializerContext.env.mode.dev;`